enki/PhysicalEngine.cpp

```diff
@@ -1180,15 +1192,24 @@ namespace Enki
 		// init non-physics interactions
 		for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
 		{
+			if(SkipPhysicsObjectsList.find(*i) != SkipPhysicsObjectsList.end())
+				continue;
+
 			(*i)->initLocalInteractions(dt, this);
 			(*i)->initGlobalInteractions(dt, this);
 		}
 
 		// interact objects together
 		for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
 		{
+			if(SkipPhysicsObjectsList.find(*i) != SkipPhysicsObjectsList.end())
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.